Core/Graphics/Source/Graphics.cpp

```diff
@@ -1,42 +1,29 @@
 #include <Babylon/Graphics.h>
+#include <Babylon/GraphicsPlatform.h>
 #include "GraphicsImpl.h"
+#include <Babylon/Graphics.h>
 
 namespace Babylon
 {
+    class Graphics::Impl : public GraphicsImpl { };
```

Comment:
What is the purpose of this?

Reply:
GraphicsImpl.h is now no longer included by Graphics.h,
So we declare Graphics::Impl in Graphics.h, then define it as our internal type in the .cpp where we use it.

The idea was to remove it from the public contract.

Reply:
Until I realized that we also need to change this for consumers of the private contract as well, so the intent of this change was wrong. I believe reverting this would be a reasonable option.